recipes/xapian/build.sh

```diff
@@ -0,0 +1,31 @@
+set -e
```

Comment:
I believe conda-build runs this as `bash -x -e build.sh`, so this shouldn't be necessary. [This](https://conda.io/docs/user-guide/tutorials/build-pkgs.html#writing-the-build-script-files-build-sh-and-bld-bat) is the closest I could find to confirmation.

Reply:
Oh yeah, that was just for my own debugging.